### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by lenafb** [hongquanli/octopi-research#10]
Would it be possible to change save settings such that .png is the default file format instead of .bmp?

**Comment by hongquanli** [comment]
Yes. You can change it here: https://github.com/hongquanli/octopi-research/blob/master/software/control/_def.py (line 15). You also copy that class definition into the configuration txt file, which will override the definition in the _def.py file.

**Comment by hongquanli** [comment]
[configuration_platereader.txt](https://github.com/hongquanli/octopi-research/files/7590518/configuration_platereader.txt)
